internal/oci/container.go

```diff
@@ -296,6 +298,16 @@ func (c *Container) CreatedAt() time.Time {
 	return c.state.Created
 }
 
+// CheckpointedAt returns the container checkpoint time
+func (c *Container) CheckpointedAt() time.Time {
+	return c.state.CheckpointedAt
+}
+
+// SetCheckpointedAt sets the time of checkpointing
+func (c *Container) SetCheckpointedAt(checkpointedAt time.Time) {
+	c.state.CheckpointedAt = checkpointedAt
+}
+
```

Comment:
I don't think we should take the CheckpointedAt pieces

Reply:
Fixed